packages/vulcan-lib/lib/modules/fragments.js

```diff
@@ -89,23 +89,23 @@ export const getDefaultFragment = collection => {
   return fragmentText ? gql`${fragmentText}` : null;
```

Comment:
Can you please rollback changes to this file? This is just a formatting thing but it bloats your PR

Reply:
The reason I ventured into this file because `string-similarity` module was not found. And to test those changes I had to revert all the way back to 1.14.0!
Maybe my way of cloning the repo wasn't proper, I don't know. :man_shrugging:

Reply:
you can just do `yarn add string-similarity` to install that module.

Reply:
`string-similarity` not found module wasn't the only issue though :sweat_smile: and it's better not to bloat this PR with unrelated changes/fixes especially that I'm unsure what caused such issues